web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx

```diff
@@ -53,11 +53,11 @@ export function CliCommand({ cliCommand, onRun, isLoading }: CliCommandProps) {
         color={shouldDisplayIsLoading ? 'text.secondary' : 'text.primary'}
         width="100%"
         css={`
+          font-family: ${props => props.theme.fonts.mono};
```

Comment:
Nit: Let's move `font-family` here to its original position to avoid introducing unnecessary file changes.